mkdocs.yml

```
@@ -0,0 +1,29 @@
+site_name: rmFakeCloud
```

```suggestion
site_name: rmfakecloud
```
Since it's stylized as `rmfakecloud` in the readme.